retrofit-converters/jaxb/src/main/java/retrofit2/converter/jaxb/JaxbConverterFactory.java

```diff
@@ -60,7 +127,14 @@ private JaxbConverterFactory(@Nullable JAXBContext context) {
       Annotation[] methodAnnotations,
       Retrofit retrofit) {
     if (type instanceof Class && ((Class<?>) type).isAnnotationPresent(XmlRootElement.class)) {
```

Comment:
Is this annotation still present for JSON? That's a little weird.

Reply:
Just discovered that MOXy supports an option for an object marshal without @XmlRootElement. Then I will set this as the default behavior for json processing, and if the user's jaxb implementation does not support it, the user should add an annotation for successful marshalling